src/web/src/components/Banner.tsx

```diff
@@ -127,25 +124,24 @@ export default function Banner() {
           v {gitInfo.version}
         </a>
       </div>
-      <Grid
-        container
-        spacing={0}
-        direction="row"
-        alignItems="center"
-        justify="center"
-        className={classes.container}
-      >
-        <Grid id="back-to-top-anchor-mobile" item xs={8}>
-          <div className={classes.icon}>
-            <ScrollAction>
-              <Fab color="secondary" aria-label="scroll-down">
-                <KeyboardArrowDownIcon className={classes.arrowDownIcon} />
-              </Fab>
-            </ScrollAction>
-          </div>
-        </Grid>
-      </Grid>
-      <div className={classes.anchor} id="back-to-top-anchor" />
+      <div className={classes.textsContainer}>
+        <Typography variant="h1" className={classes.telescopeTitle}>
+          Telescope
+        </Typography>
+        <Typography variant="h4" className={classes.quoteText}>
+          “I think one of my proudest contributions to date was for Node.js.
```

Comment:
Think we should make this an array that we can use to store as many quotes as we can find, then just pick a random one to display similar to the github contributor card as well

Reply:
Please check #1974